### PR TITLE
improving RToolbar for allowing dropdown btngrp

### PR DESCRIPTION
**Pull request by PetarT**

### CODE REVIEW

File: libraries/redcore/layouts/toolbar/button/modal.php [hunk 1]
```diff
@@ -46,9 +47,21 @@
...
-<button class="<?php echo $btnClass ?>" onclick="<?php echo $cmd ?>">
-	<?php if (!empty($iconClass)) : ?>
-		<i class="<?php echo $iconClass ?>"></i>
-	<?php endif; ?>
-	<?php echo $text ?>
-</button>
+
+<?php if ($isOption) :?>
+	<li>
+		<a href="$" class="<?php echo $btnClass ?>" onclick="<?php echo $cmd ?>">
```
Comment (Kixo) [inline libraries/redcore/layouts/toolbar/button/modal.php]: href="$"?

Comment (PetarT) [inline libraries/redcore/layouts/toolbar/button/modal.php]: Missed # :P
fixing

File: libraries/redcore/layouts/toolbar/button/standard.php [hunk 2]
```diff
@@ -48,9 +49,20 @@
...
 
-<button href="#" onclick="<?php echo $cmd ?>" class="<?php echo $btnClass ?>">
-	<?php if (!empty($iconClass)) : ?>
-		<i class="<?php echo $iconClass ?>"></i>
-	<?php endif; ?>
-	<?php echo $text ?>
-</button>
+<?php if ($isOption) :?>
+	<li>
+		<a href="$" class="<?php echo $btnClass ?>" onclick="<?php echo $cmd ?>">
```
Comment (Kixo) [inline libraries/redcore/layouts/toolbar/button/standard.php]: same here href="$"

File: libraries/redcore/layouts/toolbar/toolbar.php [hunk 3]
```diff
@@ -51,14 +52,33 @@
...
-			{
-				echo $button->render();
-			}
-			?>
-		</div>
+		<?php if ($group->isMenu()):?>
+			<div class="<?php echo $groupClass;?>">
+				<a class="btn dropdown-toggle btn-primary" data-toggle="dropdown" href="#">
+					<?php if (!empty($iconClass)):?>
+						<i class="icon <?php echo $iconClass;?>"></i>
```
Comment (Kixo) [inline libraries/redcore/layouts/toolbar/toolbar.php]: icon in class is extra and not needed

**Pull request merged**